### DISCUSSION TITLE
Sqitch docker for multiple environments like production, test etc

### DISCUSSION THREAD

**Discussion by hrkachhia** [theory/sqitch#722] [General]
Hello, 

I am using Sqitch docker image and trying to learn and create proper project structure to manage snowflake database changes. 
I wants to create project structure such a way that allow me to deploy in multiple database environments like production, test etc, 

Can you please guide me for the creation stamens to allow for deployments to multiple database environments? 

I have read about the sqitch templates but I haven't found any proper tutorial for beginner. 

Could you please guide me how to implement this with sqitch docker image? See below  reference reference  database structure 

![Screenshot 2023-03-09 at 5 33 38 PM](https://user-images.githubusercontent.com/121415907/224017872-204c4499-de02-4694-aaa3-b3fcc055e949.png)

**Comment by hrkachhia** [comment]
Could you please help how to manage multiple schema in single sqitch project and how manage both using arguments with deploy

**Comment by theory** [comment]
Presumably your test and production environments are different databases. Just use separate [target](https://sqitch.org/docs/manual/sqitch-target/) URLs for them. Something like:

``` sh
sqitch target add prod db:snowflake://xxx@yyy.us-east-1/prd_database?Driver=Snowflake;warehouse=compute_wh
sqitch target add test db:snowflake://xxx@yyy.us-east-1/tst_database?Driver=Snowflake;warehouse=compute_wh
```

Then when you deploy, use the appropriate target:

``` sh
sqitch deploy test
```
